Doc/tools/templates/indexcontent.html

```diff
@@ -1,5 +1,12 @@
-{% extends "defindex.html" %}
-{% block tables %}
+{% extends "layout.html" %}
+{%- block htmltitle -%}
+<title>{{ shorttitle }}</title>
+{%- endblock -%}
+{% block body %}
+  <h1>{{ docstitle|e }}</h1>
+  <p>
+  {% trans %}Welcome! This is the documentation for Python {{ release }}{% endtrans %}
+  </p>
```

Comment:
There are minor change. ", last updated Feb 19, 2017." part is removed.
I remove it because "Last update: " is in footer.  It's duplicated information.